autoload/ale/rename.vim

```diff
@@ -84,7 +84,7 @@ function! ale#rename#HandleTSServerResponse(conn_id, response) abort
     \       'changes': l:changes,
     \   },
     \   {
-    \       'should_save': 1,
+    \       'conn_id': a:conn_id,
     \   },
```

Comment:
Same here:


```
{
    'conn_id':  a:conn_id,
    'should_save': 1,
}
```

Reply:
Done, also added an option to disable this behavior.